sips/sip-02x/sip-02x-non-sequential-multisig-transactions.md

```diff
@@ -0,0 +1,129 @@
+# Preamble
+
+SIP Number: 02x
+
+Title: Non-sequential Multisig Transactions
+
+Author: Jeff Bencin <jbencin@hiro.so>
+
+Consideration: Technical
+
+Type: Consensus
+
+Status: Draft
+
+Created: 2023-08-30
+
+License: BSD 2-Clause
+
+Sign-off:
+
+Discussions-To: https://github.com/stacksgov/sips
+
```

Comment:
`Layer` SIP field needs to be added: https://github.com/stacksgov/sips/blob/main/sips/sip-000/sip-000-stacks-improvement-proposal-process.md.

Reply:
This is `Consensus (hard fork)` right?

Reply:
Yes.